tensorboard/webapp/reloader/reloader_component.ts

```diff
@@ -85,5 +88,7 @@ export class ReloaderComponent {
       'visibilitychange',
       this.onVisibilityChange
     );
+    this.ngUnsubscribe.next();
```

Comment:
I see this is done in the other examples of this pattern. However, I do not understand why we need to call next before we call complete.